2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_setup.py:_flush():70] setting env: {}
2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f47c9bc0940>
2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_init.py:_log_setup():303] Logging user logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_135143-239ia96r/logs/debug.log
2020-12-01 13:51:43,294 INFO    MainThread:2875 [wandb_init.py:_log_setup():304] Logging internal logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_135143-239ia96r/logs/debug-internal.log
2020-12-01 13:51:43,963 INFO    MainThread:2875 [wandb_run.py:_console_start():1218] atexit reg
2020-12-01 13:51:43,964 INFO    MainThread:2875 [wandb_run.py:_redirect():1085] redirect: SettingsConsole.WRAP
2020-12-01 13:51:43,964 INFO    MainThread:2875 [wandb_run.py:_redirect():1116] Wrapping output streams.
2020-12-01 13:51:43,964 INFO    MainThread:2875 [wandb_run.py:_redirect():1132] Redirects installed.
2020-12-01 13:51:46,915 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:51:46,919 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:11,108 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:11,111 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:11,237 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:11,244 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:11,245 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:11,248 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:11,465 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:11,468 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:13,770 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:13,776 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:13,779 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:13,784 INFO    MainThread:2875 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:52:13,784 INFO    MainThread:2875 [wandb_watch.py:watch():35] Watching
2020-12-01 13:52:15,354 INFO    MainThread:2875 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:52:22,926 INFO    MainThread:2875 [wandb_run.py:_atexit_cleanup():1188] got exitcode: 0
2020-12-01 13:52:22,926 INFO    MainThread:2875 [wandb_run.py:_restore():1160] restore
2020-12-01 13:52:22,935 INFO    MainThread:2875 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "bcd5ec498d35418c921e5195781bbac0"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 10064
      total_bytes: 10064
    }
  }
}

2020-12-01 13:52:24,938 INFO    MainThread:2875 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "38b878c68b604b9cb5ba2674114ea22c"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 13540
      total_bytes: 13540
    }
  }
}

2020-12-01 13:52:26,009 INFO    MainThread:2875 [wandb_run.py:_show_summary():1450] rendering summary
2020-12-01 13:52:26,010 INFO    MainThread:2875 [wandb_run.py:_show_files():1517] logging synced files
2020-12-01 13:52:26,079 INFO    MainThread:2875 [internal.py:handle_exit():138] Internal process exited
